packages/manager/modules/pci/src/projects/project/serving/namespace/models/models.controller.js

```diff
@@ -0,0 +1,29 @@
+import { API_STATUS, VERSION_STATUS } from './models.constants';
+
+export default class PciServingNamespaceModelsController {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    CucCloudMessage,
+  ) {
+    this.$q = $q;
```

Comment:
Are `$q`, `$translate`, `API_STATUS` and `VERSION_STATUS` needed?

Can't find any occurences.